File: app/src/main/java/com/eyeliner/eyeliner/color/picker/CustomFlag.java
package com.eyeliner.eyeliner.color.picker;

import android.content.Context;
import android.widget.TextView;

import com.eyeliner.eyeliner.R;
import com.skydoves.colorpickerview.AlphaTileView;
import com.skydoves.colorpickerview.ColorEnvelope;
import com.skydoves.colorpickerview.flag.FlagView;

public class CustomFlag extends FlagView {

    private AlphaTileView alphaTileView;

    public CustomFlag(Context context, int layout) {
        super(context, layout);
        alphaTileView = findViewById(R.id.flag_color_layout);
    }

    @Override
    public void onRefresh(ColorEnvelope colorEnvelope) {
        alphaTileView.setPaintColor(colorEnvelope.getColor());
    }
}

File: app/src/main/java/com/eyeliner/eyeliner/color/picker/CustomFlag.kt
package com.eyeliner.eyeliner.color.picker

import android.content.Context
import com.eyeliner.eyeliner.R
import com.skydoves.colorpickerview.AlphaTileView
import com.skydoves.colorpickerview.ColorEnvelope
import com.skydoves.colorpickerview.flag.FlagView

class CustomFlag(context: Context, layout: Int) : FlagView(context, layout) {

    private val alphaTileView: AlphaTileView = findViewById(R.id.flag_color_layout)

    override fun onRefresh(colorEnvelope: ColorEnvelope) {
        alphaTileView.setPaintColor(colorEnvelope.color)
    }
}